requirements.txt

```diff
@@ -5,7 +5,7 @@ certifi==2021.5.30
 charset-normalizer==2.0.4
 click==7.1.2
 clickclick==20.10.2
-connexion==2.9.0
+connexion==2.14.2
```

Comment:
Connexion `2.9.0` depends on `PyYAML < 6.0.0`, upgraded to latest compatible 2.x version which works with PyYAML 6.0.1.